### PR TITLE
fixing equation rendering in 016_smesolve-inefficient-detection.md

### PR DESCRIPTION
**Pull request by LegionAtol**
image

### CODE REVIEW

**Comment by LegionAtol** [comment]
am I missing something?

**Comment by Ericgig** [comment]
@LegionAtol 
Thank you for the contribution.
No, there is probably nothing missing. 
All our tests / actions for qutip-tutorials have been failing and no one had/took the time to fix them. So the check are failing for an unrelated reason, but we want them to pass before accepting new contributions.

**Comment by Ericgig** [comment]
@LegionAtol 
Could you merge the main branch in to rerun the tests?
They should be fixed.

**Comment by LegionAtol** [comment]
I tried with:
`git pull`
`git push origin main --force`
but nothing happens, it tells me "Everything up-to-date" (because rightly no file has been changed)
how can I do?

Thank you

**Comment by Ericgig** [comment]
Since you are working in your main branch, you could probably use `sync fork` on github, but it probably don't expect you to have pushed changes to your main branch.

To do it manually, you need to pull from the main branch of this repo:
 ```
git remote add upstream https://github.com/qutip/qutip-tutorials.git
git pull upstream main
git push origin main
```

**Comment by LegionAtol** [comment]
Thanks for the help
I pressed the Sync button and it seems like testing has started now

File: tutorials-v4/time-evolution/016_smesolve-inefficient-detection.md [hunk 1]
```diff
@@ -63,13 +63,14 @@ and $dN(t)$ is a Poisson distributed increment with $E[dN(t)] = \eta \langle a^\
 
 In QuTiP, the photocurrent stochastic master equation is written in the form:
 
-$\displaystyle d\rho(t) = -i[H, \rho] dt + \mathcal{D}[B] \rho dt 
+$$\displaystyle d\rho(t) = -i[H, \rho] dt + \mathcal{D}[B] \rho dt
```
Comment (Ericgig) [inline tutorials-v4/time-evolution/016_smesolve-inefficient-detection.md:66]: All others equations are not centered, having just that one be is strange.
```suggestion
$\displaystyle d\rho(t) = -i[H, \rho] dt + \mathcal{D}[B] \rho dt 
```

File: tutorials-v4/time-evolution/016_smesolve-inefficient-detection.md [hunk 2]
```diff
@@ -63,13 +63,14 @@ and $dN(t)$ is a Poisson distributed increment with $E[dN(t)] = \eta \langle a^\
 
 In QuTiP, the photocurrent stochastic master equation is written in the form:
 
-$\displaystyle d\rho(t) = -i[H, \rho] dt + \mathcal{D}[B] \rho dt 
+$$\displaystyle d\rho(t) = -i[H, \rho] dt + \mathcal{D}[B] \rho dt 
 - \frac{1}{2}\mathcal{H}[A^\dagger A] \rho(t) dt 
-+ \mathcal{G}[A]\rho(t) d\xi$
++ \mathcal{G}[A]\rho(t) d\xi$$
```
Comment (Ericgig) [inline tutorials-v4/time-evolution/016_smesolve-inefficient-detection.md:68]:
```suggestion
+ \mathcal{G}[A]\rho(t) d\xi$
```

**Review by Ericgig — COMMENTED** [review]
Thank you for catching the error.
Let's keep the single `$` for consistency with other equations.

**Comment by LegionAtol** [comment]
the problem is that if we use a single `$` at the beginning and end, the equation doesn't render well on the screen: 

Screenshot 2024-03-03 alle 23 22 16

**Comment by Ericgig** [comment]
I have a similar issue with `$$`:
image

The issue seems to be that the equations should be on one line.

**Comment by LegionAtol** [comment]
thanks, actually it was enough to put it on a line.
Now it's displayed like this, I think it's correct:

image

**Review by Ericgig — APPROVED** [review]
Looking good,

Thank you

**Pull request merged**